countItems.js

```diff
@@ -1,8 +1,22 @@
 const cluster = require('cluster');
 
+const { parseEnvInt } = require('./utils');
+
+const HEAP_PROFILER_INTERVAL_MS =
+    parseEnvInt(process.env.HEAP_PROFILER_INTERVAL_MS, 10 * 60 * 1000);
+const HEAP_PROFILER_PATH = process.env.HEAP_PROFILER_PATH;
+
 if (cluster.isMaster) {
+    require('./utils/heapProfiler')(
```

Comment:
Is this going to enable the heap profiler at all times indiscriminately?

Reply:
this should only be enabled when the env `ENABLE_HEAP_PROFILER` is set